torch/_dynamo/variables/constant.py

```diff
@@ -160,3 +160,6 @@ def python_type(self):
 
     def as_python_constant(self):
         return self.value
+    
+    def const_getattr(self, tx, name):
+        return getattr(self.value, name)
```

Comment:
Maybe follow the ```ConstantVariable``` way to ensure returning value is not callable:
https://github.com/pytorch/pytorch/blob/18c6ca1ee1975b89250d3d9e4f708e70d03cb255/torch/_dynamo/variables/constant.py#L66-L70